modules/apps/dynamic-data-mapping/dynamic-data-mapping-test-util/src/main/java/com/liferay/dynamic/data/mapping/test/util/DDMFormLayoutTestUtil.java

```diff
@@ -85,23 +81,21 @@ public static DDMFormLayoutColumn createDDMFormLayoutColumn(
 	public static DDMFormLayoutColumn[] createDDMFormLayoutColumns(
 		String... ddmFormFieldNames) {
 
-		return Stream.of(
-			ddmFormFieldNames
-		).map(
-			ddmFormFieldName -> {
-				DDMFormLayoutColumn ddmFormLayoutColumn =
-					new DDMFormLayoutColumn();
-
-				ddmFormLayoutColumn.setDDMFormFieldNames(
-					ListUtil.fromArray(ddmFormFieldNames));
-				ddmFormLayoutColumn.setSize(
-					DDMFormLayoutColumn.FULL / ddmFormFieldNames.length);
-
-				return ddmFormLayoutColumn;
-			}
-		).toArray(
-			DDMFormLayoutColumn[]::new
-		);
+		DDMFormLayoutColumn[] ddmFormLayoutColumns =
```

Comment:
@JanisZhang I get that all the elements seem to be with same content, so you create only one and assign it to all slots in the array. However, since the `DDMFormLayoutColumn` is mutable, let's keep the original logic, creating multiple objects.